src/components/views/dialogs/ShareDialog.js

```diff
@@ -70,9 +70,16 @@ export default class ShareDialog extends React.Component {
         this.onCopyClick = this.onCopyClick.bind(this);
         this.onLinkSpecificEventCheckboxClick = this.onLinkSpecificEventCheckboxClick.bind(this);
 
+        let permalinkCreator: RoomPermalinkCreator = null;
```

Comment:
Is the type okay in a .js file?

Reply:
The magic of Flow pretending to be TypeScript

(yes)